core/types/src/svn.rs

```diff
@@ -47,4 +60,26 @@ mod test {
 
         assert_eq!(display_string, expected_string);
     }
+
+    #[test]
+    fn isv_svn_display() {
+        let sgx_isv_svn_t = 3459;
+        let isv_svn = IsvSvn::from(sgx_isv_svn_t);
+
+        let display_string = format!("{}", isv_svn);
```

Comment:
Kind of surprised clippy didn't catch this...
```suggestion
        let display_string = format!("{isv_svn}");
```

Reply:
Done.